lib/Dialect/ArithExt/IR/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+set(LLVM_TARGET_DEFINITIONS ArithExtDialect.td)
+mlir_tablegen(ArithExtDialect.h.inc -gen-dialect-decls)
+mlir_tablegen(ArithExtDialect.cpp.inc -gen-dialect-defs)
+add_public_tablegen_target(MLIRArithExtIncGen)
+
+set(LLVM_TARGET_DEFINITIONS ArithExtOps.td)
+mlir_tablegen(ArithExtOps.h.inc -gen-op-decls)
+mlir_tablegen(ArithExtOps.cpp.inc -gen-op-defs)
+add_public_tablegen_target(MLIRArithExtOpsIncGen)
+
+add_mlir_doc(ArithExtDialect ArithExtDialect arithext/ -gen-dialect-doc --dialect=arithext)
```

Comment:
We don't need any mlir docs, since CMake is not load-bearing for generating them. Let's remove all these calls.

Reply:
ok

Reply:
- remove add_mlir_doc as per review from @Jeremy Khun in latest push.